assets/blocks/course-outline/module-block/block.json

```diff
@@ -55,7 +55,7 @@
     "customClassName": {
       "type": "string"
     },
-    "bordered": {
+    "borderedSelected": {
```

Comment:
Not critical, but I think you had renamed it to `borderedSelected` when using the other approach that wasn't boolean, right?